applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -236,28 +236,28 @@ static void stream_started_cb(struct bt_audio_stream *stream)
 {
 	int ret;
 
-	LOG_INF("Audio stream %p started", (void *)stream);
+	LOG_INF("Stream started");
```

Comment:
This could make it hard to distinguish which stream is started.
Since two ISO streams could start at the same time, and the log will print "Stream started" for twice.

Reply:
Will change back